lang/cs/Org.Apache.REEF.Network/NetworkService/NetworkService.cs

```diff
@@ -65,12 +66,13 @@ public class NetworkService<T> : INetworkService<T>
             IIdentifierFactory idFactory,
             ICodec<T> codec,
             INameClient nameClient,
-            IRemoteManagerFactory remoteManagerFactory)
+            IRemoteManagerFactory remoteManagerFactory,
+            ITcpPortProvider tcpPortProvider)
```

Comment:
Same comment as above: `NetworkService` doesn't seem to need the `ITcpPortProvider` for anything but passing it to the `IRemoteManagerFactory`. The latter was introduced to avoid that situation. Wouldn't it be easier to have the `ITcpPortProvider` injected into the constructor of `IRemoteManagerFactory`?